frontend/src/pages/Skills/SkillsContainer.tsx

```diff
@@ -1,46 +1,30 @@
 import { FC, useState, useEffect, BaseSyntheticEvent } from 'react';
 
 import { useDebounce } from 'hooks';
-import { useGetSkills, useSearchSkills } from 'api/skills';
+import { useGetSkills } from 'api/skills';
 import { formatInputValue } from 'utils/helpers/searchHelpers';
 import { ISkillsListProps } from 'types/skill';
 
 import Skills from './Skills';
 
 const SkillsContainer: FC = () => {
   const [searchInputValue, setSearchInputValue] = useState<string>('');
-  const [skillFounded, setSkillFounded] = useState<ISkillsListProps[]>([]);
-
+  const [skillFounded, setSkillFounded] = useState<ISkillsListProps[] | undefined>([]);
   const debouncedSearchValue = useDebounce(searchInputValue);
-  const { data: skillSearchResponse } = useSearchSkills(debouncedSearchValue);
-  const { data: skillsResponse } = useGetSkills();
-
+  const { data: skillsResponse, isLoading: isLoadingSkills } = useGetSkills(debouncedSearchValue);
   useEffect(() => {
-    if (debouncedSearchValue) {
-      if (
-        skillSearchResponse?.filter((skillList) => {
-          return skillList.skills.filter((skill) =>
-            skill.name.toLowerCase().includes(searchInputValue.toLowerCase()),
-          );
-        })
-      ) {
-        setSkillFounded(skillSearchResponse);
-      }
-    } else {
-      setSkillFounded([]);
-    }
-  }, [debouncedSearchValue, searchInputValue, skillSearchResponse]);
+    setSkillFounded(skillsResponse);
+  }, [skillsResponse]);
 
   const handleSearchInputChange = ({ target }: BaseSyntheticEvent) => {
-    setSearchInputValue(formatInputValue(target.value));
+    setSearchInputValue(() => formatInputValue(target.value));
```

Comment:
is it dont the same as  setSearchInputValue(formatInputValue(target.value)); ?

Reply:
yes